dbt/include/athena/macros/adapters/relation.sql

```diff
@@ -36,6 +36,14 @@
   {%- endcall %}
 {%- endmacro %}
 
+{% macro set_table_relation_schema(relation, schema) %}
+  {%- if temp_schema is not none -%}
```

Comment:
```suggestion
  {%- if schema is not none -%}
```
or could you explain how do you get temp_schema here ?